store/server/features/leads/source/mutation.ts

```
@@ -0,0 +1,215 @@
+  }
+};
+
+// --- React Query Mutation Hooks ---
+
+/**
+ * Hook for creating a source.
+ */
+export const useCreateSource = () => {
+  const queryClient = useQueryClient();
+  return useMutation(createSource, {
+    onSuccess: (data) => {
```

ESLint

[error] 'data' is defined but never used.

(@typescript-eslint/no-unused-vars)